docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -213,6 +228,85 @@ To add a Task to the end of a Mission:
 
     :::
 
+### Custom Tasks  {#custom-task}
+
+Users can create their own tasks following a specific template and place them within the **/onav_log/custom_tasks** directory.
+Restarting the UGV or dockers will trigger the system to load the custom task that was created, making it available for mission use.
+Below is a sample template file named "log_inputs.py" which is elaborated further in the following section.
+
+#### Sample Custom Task
+
+```py
+#!/usr/bin/python3
+
+from onav_custom_tasks.custom_task_base import *
+
+class CustomTask(CustomTaskBase):
+  def __init__(self):
+      super().__init__("log_inputs")
+
+  def run_task(self, goal):
+    if len(goal.strings) == 0 or len(goal.floats) == 0:
+      self._as.set_aborted()

```

Comment:
A 1-line comment here would be helpful:
"Check that the inputs are valid, else abort the task"